test/template.spec.js

```diff
@@ -292,3 +292,17 @@ test('separate loader configuration for template lang and js imports', done => {
     done()
   })
 })
+
+test('render function post loaders', done => {
+  mockBundleAndRun({
+    entry: './test/fixtures/template-post.js',
+    vue: {
+      templatePostLoaders: [
+        path.resolve(__dirname, './mock-loaders/js')
+      ]
+    }
+  }, ({ exports }) => {
+    expect(exports.staticRenderFns[0].toString()).toContain('Changed!')
```

Comment:
I'm _guessing_ this is an appropriate test. Are the static render functions invoked by the `render` function?